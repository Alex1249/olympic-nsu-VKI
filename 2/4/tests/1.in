8
4 0
4 1
4 2
4 3
4 4
10 5
100 50
1000 500
