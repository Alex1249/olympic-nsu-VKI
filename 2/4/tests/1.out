1
4
6
4
1
252
1.00891344545564e+29
2.70288240945437e+299
